contracts/src/v0.8/automation/dev/test/AutomationRegistry2_3.t.sol

```diff
@@ -766,6 +804,13 @@ contract SetConfig is SetUp {
     assertEq(transmitters, NEW_TRANSMITTERS);
   }
 
+  function _getRegistrars() private pure returns (address[] memory) {
+    address[] memory registrars = new address[](2);
+    registrars[0] = address(uint160(uint256(keccak256("registrar1"))));
+    registrars[1] = address(uint160(uint256(keccak256("registrar2"))));
+    return registrars;
+  }
```

Comment:
nitpick - but since the function is only used once, it might be neater to just inline it? not a strong opinion

Reply:
The reason of having this function is that we cant initiate and add elements to the registrars array outside of a function. If you see the original code, the registrars were under contracts.